### ISSUE TITLE
Watchparty identified as live

### ISSUE THREAD

**Issue by BarryCarlyon** [twitchdev/issues#239]
**Brief description**

Not sure if bug or intended.

But a watchparty is marked as "live" in the Helix API

**How to reproduce**

Use get streams against a stream that is in watchparty mode

**Expected behavior**

Returns `"type":"watchparty"`

**Screenshots**

![image](https://user-images.githubusercontent.com/20999/95121654-43699f00-0747-11eb-9c31-a585d8f361f6.png)

**Comment by jbulava** [comment]
I would assume this is intended, but we can confirm.  Are there issues or problems that arise in your development for a watch party being marked as live?

**Comment by BarryCarlyon** [comment]
Yes because I as a developer need to know what kind of stream it is.

I also will probably open a user voice for "get watch party episode/progress information" too. But thats not for here
